### PR TITLE
Increase icon size and allow transparent icons on action link component

### PR DESCRIPTION
**Pull request by alex-ju**
## What
Allow transparent arrow and increase icon size to 45px as requested by design.

## Why
To be used immediately on the coronavirus landing page

## Visual Changes
Screenshot 2020-09-08 at 13 36 57

[Trello card](https://trello.com/c/k7VHdLXd/792-cop-9-sept-landing-page-changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Step by step nav header updated to use heading ([PR #1671](https://github.com/alphagov/govuk_publishing_components/pull/1671))
+* Increase icon size and allow transparent icons on action link component ([PR #1674](https://github.com/alphagov/govuk_publishing_components/pull/1671))
```
Comment (andysellick) [inline CHANGELOG.md:13]: This should match the PR title (or vice versa - this seems like a good description).

File: app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path style="fill:#fff500;" d="M 17.009766,6.8183592 14.181641,9.6464838 22.035156,17.5 0.03515625,17.566406 0,21.5 h 22.035156 l -7.853515,7.853516 2.828125,2.828125 L 29.691406,19.5 Z M 39,19.5 C 39,30.269553 30.269553,39 19.5,39 9.3731566,39 1.0492998,31.280497 0.09185841,21.404918 0.03109354,20.778156 0,20.142709 0,19.5 0,8.7304474 8.7304474,0 19.5,0 30.269553,0 39,8.7304474 39,19.5 Z"/>
```
Comment (andysellick) [inline app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg:2]: Can this be compressed any further? Seems to be a couple of newlines and a tab that aren't needed? Or do we care about readability here?

**Review by andysellick — APPROVED** [review]
Looks good, just a couple of minor things. Also, curious to know if the transparent option works if the small option is also invoked? I'm aware that the options for this component are mounting, and you can specify all of them if you want to. 

Having small and transparent together is probably good, but unrelated to this I'm concerned about the resilience of the component to being passed conflicting options (not a blocker to this PR though).

File: app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg (hunk 2 above)
Comment (pixeltrix) [reply app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg:2]: Running it through https://jakearchibald.github.io/svgomg/ takes it down by 33%:

``` svg
<svg width="39" height="39" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M17.01 6.818l-2.828 2.828 7.853 7.854-22 .066L0 21.5h22.035l-7.853 7.854 2.828 2.828L29.69 19.5zM39 19.5C39 30.27 30.27 39 19.5 39 9.373 39 1.05 31.28.092 21.405A19.737 19.737 0 010 19.5C0 8.73 8.73 0 19.5 0S39 8.73 39 19.5z" fill="#fff500"/></svg>
```

Comment (alex-ju) [reply app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg:2]: This is the compressed version (in terms of polygons) – couldn't make it smaller than this, but open to suggestions. I'll remove the white space, although I'm not sure it makes a difference overall.

**Comment by alex-ju** [comment]
> Looks good, just a couple of minor things. Also, curious to know if the transparent option works if the small option is also invoked? I'm aware that the options for this component are mounting, and you can specify all of them if you want to.
> 
> Having small and transparent together is probably good, but unrelated to this I'm concerned about the resilience of the component to being passed conflicting options (not a blocker to this PR though).

Thanks a lot for the prompt review @andysellick!

Small and transparent seem to be working good together – tested now in `collections` – haven't catered for it, but will have a look after this rush and update with an example.

File: app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg (hunk 2 above)
Comment (alex-ju) [reply app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg]: Compressed further; thanks both! 🗜

**Pull request merged**